plugins/scaffolder-backend-module-http-request/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
```

Comment:
Should we include this task in the backend?

Reply:
I think we should!

Reply:
It's currently using it from source

Reply:
What do you mean it is using it from source?